CODE/vulnscan.py

```
@@ -0,0 +1,186 @@
+        vectorizer: Vectorizer to transform file content.
+        file_path (str): Path to the file to be scanned.
+
+    Returns:
+        tuple: (True if the file is sensitive, False otherwise, prediction probability).
+    """
+    mime_type, _ = mimetypes.guess_type(file_path)
+    if mime_type and mime_type.startswith('text'):
+        with open(file_path, 'r', encoding='utf-8', errors='ignore') as file:
+            content = file.read()
+        return is_sensitive(model, vectorizer, content)
+    else:
```

Ruff (0.8.0)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)